src/content/components/Router/Router.js

```diff
@@ -217,6 +218,21 @@ export class Router extends React.PureComponent {
           render: props => <Triage {...props} metas={this.props.metas} />,
         },
       },
+      {
+        label: "Release Report",
```

Comment:
can you maybe make this hidden for now until we get the query working for the release report part>

Reply:
yep